inference/worker/basic_hf_server.py

```diff
@@ -42,6 +47,51 @@ def terminate_server(signum, frame):
 
 model: transformers.PreTrainedModel
 tokenizer: transformers.PreTrainedTokenizer
+fully_loaded: bool = False
+model_input_queue: Queue = Queue()
+
+
+def model_thread():
+    request: interface.GenerateStreamRequest
+    output_queue: Queue
+    eos_token = ""
```

Comment:
Preference really but I feel a one-liner is clearer here

`eos_token = tokenizer.eos_token if hasattr(tokenizer, "eos_token") else ""`